pages/common/keepassxc-cli.md

```diff
@@ -0,0 +1,36 @@
+# keepassxc-cli
+
+> Command-line interface for KeepassXC.
+> More information: <https://keepassxc.org/>.
```

Comment:
I think we should use a link to a man page because the website doesn't document the command or it's hard to find (because I didn't find it).
```suggestion
> More information: <https://manned.org/keepassxc-cli>.
```